src/command.ts

```diff
@@ -308,3 +310,22 @@ export class Command {
     };
   }
 }
+
+// Project IDs must follow a certain format, as documented at:
+// https://cloud.google.com/resource-manager/reference/rest/v1beta1/projects#resource:-project
+// However, the regex below, matching internal ones, is more permissive so that
+// some legacy projects with irregular project IDs still works.
+const PROJECT_ID_REGEX = /^(?:[^:]+:)?[a-z0-9\-]+$/;
```

Comment:
The linter is right - I don't think you need to escape the last `-`

Reply:
Good catch! Fixed.